config/default.schema.json

```diff
@@ -274,6 +274,10 @@
 				"EXPIRES_SECONDS"
 			]
 		},
+		"COOKIE_SECRET": {
+			"type": "string",
+			"description": "Secret used to sign the session ID cookie"
+		},
```

Comment:
to not break anything (and according to Jan's comment in the ticket), this should default to 'secret'

Reply:
I talked to Jan. He agreed that it is safer to use a new password. Maybe some old sessions might invalidate, but that's not a problem.